src/components/auth/AuthGateway.tsx

```diff
@@ -0,0 +1,31 @@
+import { IPublicClientApplication } from "@azure/msal-browser";
+import { applicationConfig } from "../../config";
+import { IAuthGateway } from "./IAuthGateway";
+
+export class AuthGateway implements IAuthGateway {
```

Comment:
Just on organisation of files it might be better to keep this in the `gateways` directory where we have the other gateways.

What are your thoughts?

Reply:
Ah yeah, wasn't sure if it was more _auth_ or more _gateway_ but if it's better in _gateways_, happy to move it!